src/AppBundle/Form/PostType.php

```diff
@@ -43,13 +43,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'attr' => array('autofocus' => true),
                 'label' => 'label.title',
             ))
-            ->add('summary', 'textarea', array('label' => 'label.summary'))
-            ->add('content', 'textarea', array(
+            ->add('summary', 'Symfony\Component\Form\Extension\Core\Type\TextareaType', array('label' => 'label.summary'))
+            ->add('content', 'Symfony\Component\Form\Extension\Core\Type\TextareaType', array(
```

Comment:
@yceruto, I think it's better to add a validation rule for it, instead of using the `maxlength`